diagnostic.md

```diff
@@ -40,7 +44,7 @@ methods.
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is a recursive tree......nailed it.
```

Comment:
Ummmmmm its more of an interface that lets scripts interact with html, but it is a tree.